Assets/Scripts/MarkersController.cs

```diff
@@ -10,12 +10,12 @@ public class MarkersController : MonoBehaviour
     private Color colorOrange = new Color(255f / 255f, 106f / 255f, 0f / 255f);
     private Color colorGreen = new Color(76f / 255f, 255f / 255f, 0f / 255f);
     private Color colorBlue = new Color(0f / 255f, 148f / 255f, 255f / 255f);
-    public float markerLineWidth = .035f;
+    public float markerLineWidth = .1f;
     public bool markersVisible = true;
+    public bool poleLineVisible = true;
```

Comment:
Do we need a similar `equatorLineVisible`?  Probably worth turning it off too if the pole line is off.  Hard to see in 2D, but it will appear when the user looks around in AR/VR.

Reply:
Lots of changes made to ensure we create all markers initially, then only show the ones we want.